From: Dorota Czaplejewicz <dorota.czaplejewicz@puri.sm>
Date: Thu, 28 Apr 2022 15:28:36 +0000
Subject: glib: Fix import

Origin: upstream, https://gitlab.gnome.org/World/Phosh/squeekboard/-/commit/c0f57e735528679f2dd5c7d5acc7ea45651ca627
---
 src/panel.rs | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/panel.rs b/src/panel.rs
index 53d9559..d1af059 100644
--- a/src/panel.rs
+++ b/src/panel.rs
@@ -36,7 +36,6 @@ use crate::util::c::Wrapped;
 pub mod c {
     use super::*;
     use glib;
-    use gtk::Continue;
     use std::os::raw::c_void;
 
     use crate::outputs::c::WlOutput;
@@ -67,7 +66,7 @@ pub mod c {
         glib::idle_add_local(move || {
             let panel = panel.clone_ref();
             panel.borrow_mut().set_configured(Size{width, height});
-            Continue(false)
+            glib::Continue(false)
         });
     }
 }
